### PR TITLE
Fix some spelling issue Issue #11984

### LINKED ISSUE #11984: [Tracking] Code Spell-Checking Sprint
This is an approach suggestion for now, please feel free to comment.

Comments and variables in the source code still contain a LOT of typos or weird "spelling variations".
I would suggest, we create a list of areas (doing it all at the same time is WAY too invasive of course) for tracking, and then put the sub issues derived from that out there up-for-grabs, after we've defined, how we want to tackle this. Here is a rule list as a first suggestion:

* **Spellcheck/Formatting - don't do anything else:** Sometimes you see a "potential quick change", like lambda-ing a property getter. Resist! 😸 Let's concentrate on spellchecking and reformatting, but not refactoring, no matter how tempting!
* **(Obvious) Typos inside of comments:** Correct typos indicated by VS, but do not correct overzealously, meaning: If how an indicated typo to solve is not immediate clear to you, just leave it. If it's an impactful change (like a typo which you're not sure to correct appears 20 times in a source file), rather create a new issue.
* **Double-Space-Rule:** Eliminate double-spaces behind a terminal punctuation, since this is a) a phased-out rule, which is no longer practiced, b) was US/English only and doesn't compute for an international audience and c) gets in the way, if it is applied accidentally inside of XML comments.
* **Typos inside of local variables:** Correct obvious typos in local variables, if the names are NOT abbreviated. Also correct accidentally used `camelcAsenAming` (`camelCaseNaming`), but do not introduce camel case naming, unless you can answer "What would @JeremyKuhne or @lonitra do" confidently: The reason is, that we have a lot of Win32 API calls in the repo, and those variables have a naming standard which consist of a lot weird abbreviations and in addition is often just not camelCase, and not even camelCaseIsh.
* **Scope of other member items**: NEVER change member names, which are `protected`, `internal` (`Friend`), `virtual` (`Overridable`) let alone `public`. If there is an obvious typo in such a member, please create a new issue so we can discuss, if we want to touch it (sometimes even changing an Internal member can be a breaking change, since ... you know ... Reflection...).

- [ ] Control.cs (Sub-issue/PR)
- [ ] Application.cs (Sub-issue/PR)
- [ ] DataGridView (Sub-issue/PR)
- [ ] [...to extend...] (Sub-issue/PR)

 @Olina-Zhang, once we finalize this item and know what we want to do, we should take it over to the Designer repo, and - as suggestion - do the same?

### PR DESCRIPTION
**Pull request by paul1956**
Partly Fixes #11984

## Proposed changes
- Fix some spelling issues

## Customer Impact
- N/A

## Regression? 
- No

## Risk
- None

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12154)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/Accessibility/AccessibleObject.cs [hunk 1]
```diff
@@ -2677,7 +2677,7 @@ private VARIANT AsChildIdVariant(AccessibleObject? obj)
     /// </summary>
     private static IDispatch* GetIDispatch(AccessibleObject? obj)
     {
-        // we are always wrapping system iaccessible..
+        // we are always wrapping systemIAccessible..
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Accessibility/AccessibleObject.cs:2680]:
```suggestion
        // We are always wrapping SystemIAccessible.
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 2]
```diff
@@ -1400,7 +1400,7 @@ protected virtual CreateParams CreateParams
                 cp.Style |= (int)WINDOW_STYLE.WS_VISIBLE;
             }
 
-            // Unlike Visible, Windows doesn't correctly inherit disabledness from its parent -- an enabled child
+            // Unlike Visible, Windows doesn't correctly inherit disabled-ness from its parent -- an enabled child
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs:1403]:
```suggestion
            // Unlike Visible, Windows doesn't correctly inherit disabled state from its parent -- an enabled child
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs [hunk 3]
```diff
@@ -4432,7 +4432,7 @@ internal void BeginUpdateInternal()
     }
 
     /// <summary>
-    ///  Brings this control to the front of the zorder.
+    ///  Brings this control to the front of the zOrder.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Control.cs:4435]: Wikipedia spells it as `Z-order`

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/ComboBox/ComboBox.FlatComboAdapter.cs [hunk 4]
```diff
@@ -27,7 +27,7 @@ internal class FlatComboAdapter
 
         public FlatComboAdapter(ComboBox comboBox, bool shouldRedrawAsSmallButton)
         {
-            // adapter is re-created when combobox is resized, see IsValid method, thus we don't need to handle DPI changed explicitly
+            // adapter is re-created when comboBox is resized, see IsValid method, thus we don't need to handle DPI changed explicitly
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/ComboBox/ComboBox.FlatComboAdapter.cs:30]:
```suggestion
            // adapter is re-created when ComboBox is resized, see IsValid method, thus we don't need to handle DPI changed explicitly
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs [hunk 5]
```diff
@@ -859,7 +859,7 @@ public event EventHandler? SelectedIndexChanged
     }
 
     /// <summary>
-    ///  Occurs before a tabpage is selected as the top tabPage.
+    ///  Occurs before a tabPage is selected as the top tabPage.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs:862]:
```suggestion
    ///  Occurs before a tabPage is selected as the top TabPage.
```

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs [hunk 6]
```diff
@@ -870,7 +870,7 @@ public event TabControlCancelEventHandler? Selecting
     }
 
     /// <summary>
-    ///  Occurs after a tabpage is selected as the top tabPage.
+    ///  Occurs after a tabPage is selected as the top tabPage.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/TabControl/TabControl.cs:873]:
```suggestion
    ///  Occurs after a TabPage is selected as the top TabPage.
```

THere are more occurrences of camel-casing - tabPage in this, please spell them as TabPage.

File: src/System.Windows.Forms/src/System/Windows/Forms/Input/InputLanguage.cs [hunk 7]
```diff
@@ -109,8 +109,11 @@ public string LayoutName
     }
 
     /// <summary>
-    ///  Returns the <see href="https://learn.microsoft.com/windows/win32/api/winuser/nf-winuser-getkeyboardlayoutnamew">
-    ///  keyboard layout identifier</see> of the current input language.
+    ///  Returns the
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Input/InputLanguage.cs:112]: I don't this this change improves readability, please revert it and the next one on line 180, this looks like a bad merge.

File: src/System.Windows.Forms/src/System/Windows/Forms/Control.cs (hunk 3 above)
Comment (paul1956) [reply src/System.Windows.Forms/src/System/Windows/Forms/Control.cs:4435]: Fixed

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/ComboBox/ComboBox.FlatComboAdapter.cs (hunk 4 above)
Comment (paul1956) [reply src/System.Windows.Forms/src/System/Windows/Forms/Controls/ComboBox/ComboBox.FlatComboAdapter.cs:30]: Fixed

File: src/System.Windows.Forms/src/System/Windows/Forms/Input/InputLanguage.cs (hunk 7 above)
Comment (paul1956) [reply src/System.Windows.Forms/src/System/Windows/Forms/Input/InputLanguage.cs:112]: @Tanya-Solyanik this change was from @ricardobossan PR feedback, all See href were changed to this format. It shortens the long lines a lot.

I can revert them all but most aren't my changes.

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/Labels/Label.cs [hunk 8]
```diff
@@ -124,7 +124,7 @@ public override bool AutoSize
     }
 
     /// <summary>
-    ///  This property controls the activation handling of bleedover for the text that
+    ///  This property controls the activation handling of bleed over for the text that
     ///  extends beyond the width of the label.
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Controls/Labels/Label.cs:128]: Could you please use the test from the docs instead? This comment is not helping to understand what this property is doing.
```
Gets or sets a value indicating whether the ellipsis character (...) appears at the right edge of the Label, denoting that the Label text extends beyond the specified length of the Label.
```

Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/src/System/Windows/Forms/Controls/Labels/Label.cs:128]: If prefer to do it in the next PR, let me know.

**Review by Tanya-Solyanik — APPROVED** [review]

File: src/System.Windows.Forms/src/System/Windows/Forms/Controls/Labels/Label.cs (hunk 8 above)
Comment (paul1956) [reply src/System.Windows.Forms/src/System/Windows/Forms/Controls/Labels/Label.cs:128]: I will do in a separate PR thanks. This text was what was there all I did was fix spelling issue.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**